specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -17129,6 +17129,30 @@
           "items": {
             "$ref": "#/definitions/IPConfig"
           }
+        },
+        "recoveryNicName": {
+          "description": "The name of the NIC to be used when creating target NICs.",
+          "type": "string"
```

Comment:
adding new fields to response in existing api version is breaking change.
Refer to versioning in [https://armwiki.azurewebsites.net/api_contracts/Intro.html](https://armwiki.azurewebsites.net/api_contracts/Intro.html)

Let us know how the service is going to ensure compatibility with existing clients/sdks using the older swagger.